NEWS.md

```diff
@@ -93,6 +93,7 @@ function calls. (#850, #851, @renkun-ken)
   * Each linter can have multiple tags
   * New function `available_linters()` to list available linters and their tags 
     This feature is extensible by package authors providing add-on linters for {lintr}.
+  * New function `linters_with_tags()` to help build a list of linters using tags.
```

Comment:
The NEWS should mention the supersession plan? Or we save that until #1029 is done?

Reply:
Best done in tandem with #1029 I think, when we know what the final outcome of the refactoring will be.